common/src/address.rs

```diff
@@ -88,36 +88,36 @@ lazy_static::lazy_static! {
 
     /// The IPv4 subnet for External DNS OPTE ports.
     pub static ref DNS_OPTE_IPV4_SUBNET: Ipv4Net =
-        Ipv4Net(Ipv4Network::new(Ipv4Addr::new(172, 30, 0, 0), 24).unwrap());
```

Comment:
What was the motivation for starting at `1`?

Reply:
[RFD 21](https://rfd.shared.oxide.computer/rfd/0021#concept-subnet) says every VPC gets a default subnet which would be the first subnet. We don't make use of it for services but I just shifted these all over in case we wanted to use that first default subnet for cross-service traffic over opte in the future.